projects/plugins/jetpack/modules/widgets/class-jetpack-eu-cookie-law-widget.php

```diff
@@ -222,20 +227,21 @@ public function form( $instance ) {
 					'modules/widgets/eu-cookie-law/eu-cookie-law-admin.js'
 				),
 				array( 'jquery' ),
-				20180417
+				20180417,
+				false
 			);
 
-			require( dirname( __FILE__ ) . '/eu-cookie-law/form.php' );
+			require __DIR__ . '/eu-cookie-law/form.php';
 		}
 
 		/**
 		 * Sanitize widget form values as they are saved.
 		 *
 		 * @param  array $new_instance Values just sent to be saved.
-		 * @param  array $old_instance Previously saved values from database.
+		 * @param array $old_instance Previously saved values from database.
```

Comment:
Probably both lines should have either one space or two between `@param` and the type.

I'd do one space, but I don't really care which any more than phpcs does.